lib/src/sql/value/value.rs

```diff
@@ -2940,8 +2940,8 @@ mod tests {
 	#[test]
 	fn check_size() {
 		assert_eq!(64, std::mem::size_of::<Value>());
-		assert_eq!(104, std::mem::size_of::<Error>());
-		assert_eq!(104, std::mem::size_of::<Result<Value, Error>>());
+		assert_eq!(136, std::mem::size_of::<Error>());
+		assert_eq!(136, std::mem::size_of::<Result<Value, Error>>());
```

Comment:
It's important that the `Error` type is as small as possible for memory usage reasons when dealing with query responses. This is the reason that we convert most values to `String`. We actually want to reduce the size of `Error` below `104` rather than increase it. Therefore we would need to change the `Error::PatchTest` implementation so that it reduces the size. Then these line changes can be reverted.